Week 3/NomadBusinessCard/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,138 @@
+
+    <ImageView
+        android:id="@+id/imageView2"
+        android:layout_width="192dp"
+        android:layout_height="258dp"
+        android:contentDescription="@string/todo"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.985"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0"
+        app:srcCompat="@drawable/maasai" />
+
+    <TextView
+        android:id="@+id/quotespace"
+        android:layout_width="382dp"
+        android:layout_height="113dp"
+        android:fontFamily="cursive"
+        android:text="@string/facts"
+        android:textAlignment="center"
+        android:textColor="#070707"
+        android:textSize="24sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.551"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageView"
+        app:layout_constraintVertical_bias="0.002" />
+
+    <Button
+        android:id="@+id/quotegenerator"
+        android:layout_width="125dp"
+        android:layout_height="42dp"
+        android:background="@drawable/roundbutton"
+        android:text="@string/let_s_have_fun"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/quotespace"
+        app:layout_constraintVertical_bias="0.1" />
+
+    <TextView
+        android:id="@+id/callme"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/call_us"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/quotegenerator"
+        app:layout_constraintVertical_bias="0.231" />
+
+    <TextView
+        android:id="@+id/emailme"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/email_us"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.514"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/callme"
+        app:layout_constraintVertical_bias="0.188" />
+
+
+    <ImageButton
+        android:id="@+id/imageButton2"
```

Comment:
It would be a good idea to use more descriptive names as an id to controls on the layout.